src/kaocha/api.clj

```diff
@@ -1,3 +1,4 @@
+(require 'kaocha.version_check)
```

Comment:
can we use `version-check` instead of `version_check`? I think that would be more idiomatic and future-proof.

This "require before ns" trick is interesting. I'm sure some static analysis tools are not going to like it, but I think it's fine. We'll see if anyone complains.